definitions/npm/redux-devtools-extension_v2.x.x/flow_v0.104.x-/redux-devtools-extension_v2.x.x.js

```diff
@@ -94,7 +127,7 @@ declare function $npm$ReduxDevtoolsExtension$composeWithDevTools<A, B, C, D, E,
   cd: C => D,
   bc: B => C,
   ab: A => B
-): A => H;
+): A => I;
```

Comment:
Did we determine if we still needed this after the `$Compose` change above?

Reply:
I have unless you disagree, regardless of the $Compose change above, I think this return type definition is wrong.

Reply:
I don't think the return type change is wrong. I'm just wondering if we need the overloads since `$Compose` should cover all arities completely.